.gitignore

```diff
@@ -2,7 +2,6 @@
 /.*
 !.gitignore
 !/.unicompilerc.json
-!/.env.example
```

Comment:
Don't remove this. This line negates the pattern on the second line.

Line 2 says: ignore all files beginning with `.`
Line 5 says: don't ignore `.env.example`